src/components/password/password.stories.mdx

```diff
@@ -0,0 +1,165 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import LinkTo from "@storybook/addon-links/react";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import Password from ".";
+import * as stories from "./password.stories.tsx";
+
+<Meta title="Password" parameters={{ info: { disable: true } }} />
+
+# Password
+
+Provide a way for the user to securely enter a password.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+```javascript
+import Password from "carbon-react/lib/components/password";
+```
+
+## Examples
+
+### Default
+
+<Canvas>
+  <Story name="default" story={stories.Default} />
+</Canvas>
+
+### With other input components used to change type
```

Comment:
```suggestion
### Controlling visibility of password
```
What do you think about this wording tweak? This might make it clearer about what this pattern is about at a glance from a design perspective